.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
@ljharb IMHO "node" is even more ambiguous than "stable". As far I can see, "node" could read as any Node.js version.